terraform/stacks/main/stack.tf

```
@@ -305,6 +305,15 @@ module "diego" {
   log_bucket_name = module.log_bucket.elb_bucket_name
 }
 
+module "opensearch_logs_customer" {
+  count                = var.deploy_opensearch_logs_customer ? 1 : 0
+  source               = "../../modules/opensearch_domain"
+  private_elb_subnets  = [module.cf.services_subnet_az1, module.cf.services_subnet_az2]
+  domain_name          = "{var.stack_description}-logs-customer"
+  master_user_name     = "{var.master_user_name}"
+  master_user_password = "{var.master_user_password}"
```

```suggestion
  master_user_name     = var.opensearch_logs_customer_master_username
  master_user_password = var.opensearch_logs_customer_master_password
```

While this is passing terraform validation, it won't do what you expect on `terraform plan`. You're interpolating a variable of `master_user_name` here, but there is no variable defined in `terraform/stacks/main/variables.tf` with that name. 

So we should add these variables to that file:

- `opensearch_logs_customer_master_username`
- `opensearch_logs_customer_master_password`